pom.xml

```diff
@@ -110,21 +111,10 @@
 		<dependency>
 			<groupId>org.janelia.saalfeldlab</groupId>
 			<artifactId>n5</artifactId>
-			<version>${n5.version}</version>
+			<version>${org.janelia.saalfeldlab.n5.version}</version>
```

Comment:
Do we still need different test and build versions or is that resolved?

Reply:
I'll defer @ctrueden to on this.

Reply:
Nah, @ctrueden is busy, I will check it.  We had a phase n5-hdf5 requiring an older version of n5 for tests than the current one it was built against, this is probably obsolete now.

Reply:
Oh, this is for the test to build with the inherited test classes available. Sorry for the noise, it's fine!